src/utils/media-devices-manager.js

```diff
@@ -86,7 +92,7 @@ export default class MediaDevicesManager extends EventEmitter {
         return this._micDevices;
       }
     } else {
-      return [{ value: DEFAULT_DEVICE_ID, label: "Default" }];
+      return [{ value: this.defaultInputDeviceId, label: this.micLabelForDeviceId(this.defaultInputDeviceId) }];
```

Comment:
Its a bit hard to trace through but I think this branch will never actually be hit where `defaultInputDeviceId` will return anything but 'none' because `this._micDevices` will never be set since its guarded around the same condition https://github.com/mozilla/hubs/pull/5230/files#diff-68c343bc022cf6fd909117f937d2d42768cee117b0d0649e9ffc77bd22b83c78R237 (you don't actually see the relationship here because its hidden behind the getter.

I think this whole block can be simplified into:
```js
return MediaDevicesManager.isAudioInputSelectEnabled && this._permissionsStatus[MediaDevices.MICROPHONE] !== PermissionStatus.DENIED ?
         this._micDevices :
         [{ value: NO_DEVICE_ID, label: "None" }];
```

Reply:
Yep, true again. Simplifying.